### DISCUSSION TITLE
hierarchy view of neo4j

### DISCUSSION THREAD

**Discussion by Dileep17** [neo4j-contrib/neovis.js#323] [Q&A]
Hi, I have hierarchy information stored in neo4j. Im trying to display nodes in the hierarchy view. I'm not able to get it working or able to find an example. 

It would be of great help if some examples of hierarchy display can be provided.

**Comment by Dileep17** [comment]
Anyone able to be a hierarchy view working with [neovis.js](https://github.com/neo4j-contrib/neovis.js) ?

**Comment by thebestnom** [comment]
https://github.com/neo4j-contrib/neovis.js/blob/master/MIGRATING.md
There is directed graph in the migration guide
